mozilla-central/reblame

```diff
@@ -9,5 +9,5 @@ set -o pipefail # Check all commands in a pipeline
 # as to minimize the amount of new work needed. The order of branches
 # in the loop is also selected to reduce unnecessary work; changing the
 # order should not affect correctness but may increase redundant work.
-BRANCHES="beta release esr78 esr68 esr60 esr45 esr31 esr17"
+BRANCHES="beta release esr78 esr68 esr60 esr45 esr31 esr17 pine"
```

Comment:
I would put `pine` at the start of this list since it's probably pretty close to m-c. Putting it at the end means the code will start with the esr17 blame branch and build up pine from there, which will be a lot more work.